core/src/test/java/org/apache/iceberg/TestBase.java

```diff
@@ -643,6 +666,22 @@ protected DataFile newDataFile(String partitionPath) {
         .build();
   }
 
+  protected DeleteFile fileADeletes() {
+    return formatVersion >= 3 ? FILE_A_DV : FILE_A_DELETES;
+  }
+
+  protected DeleteFile fileBDeletes() {
+    return formatVersion >= 3 ? FILE_B_DV : FILE_B_DELETES;
+  }
+
+  protected DeleteFile newDeletes(DataFile dataFile) {
```

Comment:
maybe `newDeleteFile`?

Reply:
Other `newDeleteFile` methods below return V2 position deletes, so I wanted to differentiate. I am OK renaming too.

Reply:
I believe callers won't care as long as you get a correct `DeleteFile` for the correct `formatVersion`, so I was thinking that we most likely don't need to differentiate here with the naming

Reply:
I have no preference, to be honest. I'll update in the next round.

Reply:
I went to rename and I remembered why I did it this way. I wanted a way to construct a V2 delete file independently of the format version and to not need to update the other methods that create delete files now.